### DISCUSSION TITLE
Error while running

### DISCUSSION THREAD

**Discussion by AngelicaScamarcia** [sofa-framework/sofa#3719] [Build / Config / Environment] [answered]
Hi there, 
I have this problem when I try to run a simulation:
![image](https://user-images.githubusercontent.com/95097437/224654085-fe420e55-575c-4b52-b65e-a3de9a4a3f51.png)

I have installed Python 3.8
PYTHON_ROOT:  C:\Users\scama\AppData\Local\Programs\Python\Python38
PYTHONPATH:    C:\Users\scama\SOFA\SOFA_v22.12.00_Win64\plugins\SofaPython3\lib\python3

Someone knows how can I solve it?
Thank you

**Comment by alxbilger** [comment, marked as answer]
As the error message suggests, you need to load the SofaPython3 plugin to be able to load Python scenes.
If you use `runSofa`, try `runSofa -l SofaPython3`.
Other methods are available: https://www.sofa-framework.org/community/doc/plugins/what-is-a-plugin/#plugin-loading

**Reply by AngelicaScamarcia** [reply]
Thank you, it helps but now I have this error
![image](https://user-images.githubusercontent.com/95097437/224689425-d898c09e-bc5d-45dd-81df-5dde5ebd64dc.png)

**Reply by alxbilger** [reply]
The error comes from your Python script. It tries to import SOFA, but the right syntax is `import Sofa`.

**Reply by AngelicaScamarcia** [reply]
It works, thank you so much!
